src/wr_logic_teleop_science/config/scienceLogic.yaml

```diff
@@ -0,0 +1,5 @@
+turnTablePositions:
+  - .5
+  - .7
+  - .8
+  - .9
```

Comment:
These numbers will be integers between 0 and ~1800.  While these are currently unknowable, numbers in this range will be more useful to whoever is doing initial setup.